pkgs/applications/editors/emacs-modes/emacspeak/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, rsync, gnused, emacs, tcl, tclx, espeak-ng }:
+
+stdenv.mkDerivation rec {
+  pname = "emacspeak";
+  version = "51.0";
+
+  src = fetchFromGitHub {
+    owner = "tvraman";
+    repo = "emacspeak";
```

Comment:
In Archlinux they seem to use the release tarball.
This should contain less irrelevant files: 

https://github.com/tvraman/emacspeak/blob/master/Makefile#L74
https://github.com/tvraman/emacspeak/releases/download/51.0/emacspeak-51.0.tar.bz2
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=emacspeak